src/tutorials/chat_commands_tutorial.md

```diff
@@ -0,0 +1,2146 @@
+---
+id: commands_tutorial
+name: Chat Commands
+title: Chat Commands
+tags:
+    - Tutorial
+---
+
+# Chat Commands
+
+## Overview
+
+In this tutorial you are going to create chat commands that can be typed into the chat that will perform an action if the player has permission, by using [chat hooks](../api/chat.md). You will be creating some general commands that can be useful for most games. You will also be creating commands that are aimed specifically at some of the frameworks to show how commands can be used to help test your game.
+
+* **Completion Time:** ~1 hour
+* **Knowledge Level:** It is recommended to have completed the [Scripting Beginner](lua_basics_helloworld.md) and [Scripting Intermediate](lua_basics_lightbulb.md) tutorials.
+* **Skills you will learn:**
+    * Requiring the Command Parser API.
+    * Creating permissions.
+    * Writing custom commands.
+
+---
+
+## Import Asset from Community Content
+
+You will be importing an asset from **Community Content** that will help you create the admin commands by providing you an API to speed up development.
+
+1. Open the **Community Content** window.
+2. Search for `Chat Commands` by **CoreAcademy**.
+3. Click **Import**.
+
+![!Community Content](../img/ChatCommands/cc_asset.png){: .center loading="lazy" }
+
+### Requiring the Command Parser API
+
+To create commands, you will need to require the **CommandParser** script so you have access to all the functions in the API.
+
+#### Create CommandParser Custom Property
+
+Drag and drop the **CommandParser** script on to the script you want to use to create commands.
+
+!!! tip "Context"
+    The script that you will be using to create the commands should be in a server context. If you need to create client side commands, then it is recommended to create a separate script. In a later section, server vs client commands will be discussed.
+
+![!Require Script](../img/ChatCommands/require.png){: .center loading="lazy" }
+
+#### Require the CommandParser API
+
+By requiring the Command Parser API at the top of your script, you will then have access to the functions and permissions. By default the Command Parser comes with the `promote` and `demote` commands, these can be found in the **CommandParser** script.
+
+```lua
+local CommandParser = require(script:GetCustomProperty("CommandParser"))
+```
+
```

Comment:
May be good to mention that Player Storage must be enabled for the CommandParser to work.

Reply:
The tutorial is aimed at creators that are likely adding commands to an existing game and have experience with storage, or know where the setting is.

It couldn't hurt to add a mention anyway, so will get that added.